solutions/product_solutions/NetInsight_Nimbra_Vision/01_netinsight-nimbra-vision-install.md

```diff
@@ -0,0 +1,68 @@
+---
+uid: netinsight-nimbra-vision-install
+---
+
+# Installing the DataMiner NetInsight Nimbra Vision solution
+
+## Prerequisites
+
+- DataMiner license **Tier 2** or subscription
+- DataMiner minimum version **10.4.8** or higher
+- Soft-launch features: [GenericInterface](xref:Overview_of_Soft_Launch_Options#genericinterface)
+- [NetInsight Nimbra Vision DataMiner connector](https://catalog.dataminer.services/details/e48af0b9-b52c-4106-b0e0-22c44ead85f5)
+- The necessary **Nimbra Vision Licenses**:
+
+  - **Service Provisioning**: To provision services from Nimbra Vision.
+  - **Northbound Provisioning**: For access to the REST APIs.
+  - **Scheduling**: To schedule and book resources needed by the services, e.g. ports and trunk.
+capacity.
+  - **Autoroute**: To automatically create the path based on time, and to book available resources.
+
+## Deploying the DataMiner NetInsight Nimbra Vision DataMiner solution
+
+1. Look up the [*NetInsight Nimbra Vision* package](https://catalog.dataminer.services/details/d9ec570f-a625-40c1-a6fa-c9b4f15416cd) in the Catalog.
+
+1. Click the *Deploy* button.
+
+1. Select the target DataMiner System and confirm the deployment. The package will be pushed to the DataMiner System.
+
+> [!NOTE]
+> When you download the *NetInsight Nimbra Vision* package from the Catalog, the necessary connector is included and do not need to be installed separately.
```

Comment:
If the connector is included in the package, why do you also mention it in the prerequisites? It seems better to me to remove it there.